readme/usage.md

```diff
@@ -85,7 +85,17 @@ Examples:
 - Alphanumeric `a` to `z` and `0` to `9`
 - Modifiers `Alt_L` `Control_L` `Control_R` `Shift_L` `Shift_R`
 - Mouse buttons `BTN_LEFT` `BTN_RIGHT` `BTN_MIDDLE` `BTN_SIDE` ...
-- Multimedia keys `KEY_NEXTSONG` `KEY_PLAYPAUSE` ...
+- Mouse scroll `wheel(down, 1)` `wheel(up, 1)`
+- Mouse move `mouse(left, 1)` `mouse(right, 1)` `mouse(up, 1)` `mouse(down, 1)`
+- Multimedia keys `KEY_NEXTSONG` `KEY_PLAYPAUSE` `XF86AudioMicMute` ...
+
+## Keyboard/Mouse cooperation
+
+Left click mapped to a keyboard key can be held while moving the cursor by
+a real mouse/trackball, resulting in proper drag'n'drop.
+
+Mouse scroll up/down mapped to keyboard keys can add scroll support to cheap
+3-button mice and trackballs without a scroll ring.
```

Comment:
Or gamepads, foot pedals, any device. By showing the possibility to map those kinds of events with key-mapper in the example section it should be clear that it can be added to any device. Not sure if this addition adds value